BE/src/main/resources/schema.sql

```diff
@@ -0,0 +1,30 @@
+DROP TABLE IF EXISTS dish;
+CREATE TABLE dish
+(
+    dish_id       BIGINT       NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    name          VARCHAR(255) NULL,
+    description   VARCHAR(255) NULL,
+    normal_price  DECIMAL      NULL,
+    sale_price    DECIMAL      NULL,
+    badge         VARCHAR(255) NULL,
+    delivery_type VARCHAR(255) NULL,
+    thumbnail     VARCHAR(255) NULL,
+    category_id   BIGINT       NOT NULL,
+    dish_status   VARCHAR(255) NULL
```

Comment:
`nit` 커멘트를 남길 수도 있답니다.
```suggestion
    dish_status   VARCHAR(255) NULL COMMENT '커멘트를 남길 수 있습니다. 이런 케이스가 있고 저런 케이스가 있습니다.',
```

Reply:
`COMMENT` 를 통해 어떠한 `column` 인지 추가하도록 하겠습니다. 감사합니다 ㅎㅎ